diff --git a/node_modules/expo-barcode-scanner/ios/EXBarCodeScanner/EXBarCodeCameraRequester.m b/node_modules/expo-barcode-scanner/ios/EXBarCodeScanner/EXBarCodeCameraRequester.m
index 7c9efd9..c43ac5a 100644
--- a/node_modules/expo-barcode-scanner/ios/EXBarCodeScanner/EXBarCodeCameraRequester.m
+++ b/node_modules/expo-barcode-scanner/ios/EXBarCodeScanner/EXBarCodeCameraRequester.m
@@ -18,9 +18,8 @@ - (NSDictionary *)getPermissions
   AVAuthorizationStatus systemStatus;
   UMPermissionStatus status;
   NSString *cameraUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSCameraUsageDescription"];
-  NSString *microphoneUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSMicrophoneUsageDescription"];
-  if (!(cameraUsageDescription && microphoneUsageDescription)) {
-    UMFatal(UMErrorWithMessage(@"This app is missing either 'NSCameraUsageDescription' or 'NSMicrophoneUsageDescription', so audio/video services will fail. Add both of these entries to your bundle's Info.plist."));
+  if (!(cameraUsageDescription)) {
+    UMFatal(UMErrorWithMessage(@"This app is missing 'NSCameraUsageDescription' Add entry to your bundle's Info.plist."));
     systemStatus = AVAuthorizationStatusDenied;
   } else {
     systemStatus = [AVCaptureDevice authorizationStatusForMediaType:AVMediaTypeVideo];
