diff --git a/node_modules/react-native-background-fetch/android/build.gradle b/node_modules/react-native-background-fetch/android/build.gradle
index e4470f3..0c0df0c 100644
--- a/node_modules/react-native-background-fetch/android/build.gradle
+++ b/node_modules/react-native-background-fetch/android/build.gradle
@@ -27,5 +27,5 @@ repositories{
 
 dependencies {
     implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-    implementation(group: 'com.transistorsoft', name:'tsbackgroundfetch', version: '+')
+    api fileTree(include: ['*.aar'], dir: 'libs/com/transistorsoft/tsbackgroundfetch/0.4.5')
 }
